scss/_accordion.scss

```diff
@@ -58,39 +56,36 @@
 }
 
 .accordion-item {
+
```

Comment:
need to comment bootstrap initial values and add a comment to explain its a boosted mod : 
Bootstrap initial values : 
```
margin-bottom: -$accordion-border-width;
  background-color: $accordion-bg;
  border: $accordion-border-width solid $accordion-border-color;
```